packages/monorepo-scripts/src/utils/discord.ts

```diff
@@ -0,0 +1,27 @@
+import { fetchAsync } from '@0x/utils';
+
+import { constants } from '../constants';
+
+import { utils } from './utils';
+
+export const alertDiscord = async (releaseNotes: string): Promise<boolean> => {
+    const webhookUrl = constants.discordAlertWebhookUrl;
+    if (!webhookUrl) {
+        utils.log('Not alerting to discord because webhook url not set');
+        return false;
```

Comment:
should we throw these as errors and just have a return type of `void`?

Reply:
Yes -- that seems slightly better than the current approach